apps/wpcom-block-editor/src/common/tracking/track-record-event.js

```diff
@@ -1,20 +1,38 @@
 /**
  * External dependencies
  */
-import { registerPlugin } from '@wordpress/plugins';
 import { isObjectLike, isUndefined, omit } from 'lodash';
 import debug from 'debug';
+
 const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
 
-// In case Tracks hasn't loaded
+// In case Tracks hasn't loaded.
 if ( typeof window !== 'undefined' ) {
 	window._tkq = window._tkq || [];
 }
 
 // Adapted from the analytics lib :(
 // Because this is happening outside of the Calypso app we can't reuse the same lib
 // This means we don't have any extra props like user
-const tracksRecordEvent = function( eventName, eventProperties ) {
+
+export default ( eventName, eventProperties ) => {
+	if (
+		undefined === window ||
+		undefined === window._currentSiteId ||
+		undefined === window._currentSiteType
+	) {
+		return;
+	}
+
+	// Required by Tracks when added manually
+	const blog_id = window._currentSiteId;
+	const site_type = window._currentSiteType;
+
+	// Keep tracking to Simple and Atomic sites for now.
+	if ( -1 === [ 'simple', 'atomic' ].indexOf( site_type ) ) {
+		return;
+	}
```

Comment:
This is great, but for an early return move this to top of `common/tracking.js` https://github.com/Automattic/wp-calypso/pull/34655/files#diff-b44c241cc2dca207ed1c179a99ba3c12